buildSrc/src/main/kotlin/Dependency.kt

```diff
@@ -146,8 +147,8 @@ object Dependency {
     }
 
     object Play {
-        private const val version = "18.0.0"
+        private const val version = "21.0.1"
 
-        const val location = "com.google.android.gms:play-services-location:$version"
+        const val location = "com.google.android.gms:play-services-location:18.0.0"
```

Comment:
You've updated the constant but then made it not used? This does not update gms location